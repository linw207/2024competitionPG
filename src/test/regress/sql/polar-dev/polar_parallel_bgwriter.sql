-- Test parallel background writer.
--
-- alter parallel bgwriter workers to a big value.
alter system set polar_parallel_bgwriter_workers = 20;

alter system set polar_parallel_bgwriter_workers = 8;
select pg_reload_conf();
select pg_sleep(1);

-- alter parallel bgwriter workers to a small values.
alter system set polar_parallel_bgwriter_workers = 0;
select pg_reload_conf();
select pg_sleep(1);

alter system set polar_parallel_bgwriter_workers = 3;
select pg_reload_conf();
select pg_sleep(1);