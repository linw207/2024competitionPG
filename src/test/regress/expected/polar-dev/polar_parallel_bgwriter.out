-- Test parallel background writer.
--
-- alter parallel bgwriter workers to a big value.
alter system set polar_parallel_bgwriter_workers = 20;
ERROR:  20 is outside the valid range for parameter "polar_parallel_bgwriter_workers" (0 .. 8)
alter system set polar_parallel_bgwriter_workers = 8;
select pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

select pg_sleep(1);
 pg_sleep 
----------
 
(1 row)

-- alter parallel bgwriter workers to a small values.
alter system set polar_parallel_bgwriter_workers = 0;
select pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

select pg_sleep(1);
 pg_sleep 
----------
 
(1 row)

alter system set polar_parallel_bgwriter_workers = 3;
select pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

select pg_sleep(1);
 pg_sleep 
----------
 
(1 row)

